wp-cli-templates/block-index-js.mustache

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { ExternalLink, Path } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import renderMaterialIcon from '../../utils/render-material-icon';
+import edit from './edit';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = '{{ slug }}';
+export const title = __( '{{ title }}', 'jetpack' );
+export const settings = {
+	title,
+
+	description: (
+		<Fragment>
+			<p>{ __( '{{ description }}', 'jetpack' ) }</p>
+			<ExternalLink href="#">{ __( 'Learn more about {{ title }}', 'jetpack' ) }</ExternalLink>
+		</Fragment>
+	),
+
+	/* @TODO Add the icon. You can use one of these https://material.io/tools/icons/?style=outline */
+	icon: renderMaterialIcon(
+		<Path d="M9 15h2V9H9v6zm1-10c-.5 0-1 .5-1 1s.5 1 1 1 1-.5 1-1-.5-1-1-1zm0-4c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9zm0 16c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z" />
+	),
+
+	category: 'jetpack',
+
+	keywords: [ {{#keywords}}_x( '{{ keyword }}', 'block search term', 'jetpack' ), {{/keywords}} ],
+
+	supports: {
+		html: false,
+	},
+
+	/* @TODO Write the block editor UI */
+	edit,
+
+	/* @TODO Write the block editor output */
+	save: () => <p>Block output goes here</p>,
```

Comment:
Might leave this with just the `TODO` comment. Outputting unlocalized test is undesirable.

```suggestion
	save: () => null,
```

Reply:
🆗